gap/base/recognition.gi

```diff
@@ -625,20 +652,22 @@ InstallGlobalFunction( RecogniseGeneric,
         SetParentRecogNode(riker,ri);
         Info(InfoRecog,2,"Back from kernel (depth=",depth,").");
 
+        if not IsReady(riker) then
```

Comment:
Here the same, maybe we should print something here?

Reply:
We don't need to print something here, since we're only "exiting". The warning message was already printed in one of the "failed cases" that is in the "unfinished case" part or in the "recognise the image" part.